app/metrics.yaml

```diff
@@ -4697,3 +4697,99 @@ awesomebar:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2021-08-01"
+
+android_keystore_experiment:
+  experiment_failure:
+    type: event
```

Comment:
`Actions.EXPERIMENT, Values.FAIL` fact comes with an exception too, let's record that in an `extra` like you do for other failures below.

Reply:
Thank you!
Updated the patch, tested locally this event with one IAE thrown in AC and I got
> {
          "category": "android_keystore_experiment",
          "extra": {
            "failure_exception": "java.lang.IllegalArgumentException"
          },
          "name": "experiment_failure",
          "timestamp": 15
        },